std_msgs/Header header

geometry_msgs/Polygon[] rects
int8[] labels
geometry_msgs/Point32[] world_coor